CMakeLists.txt

```diff
@@ -144,27 +144,32 @@ else()
 endif()
 
 
-#
-# For cuda we want to be able to control which architectures we compile for on 
-# a per-file basis in order to cut down on compile time. So here we extract
-# the set of architectures we want to compile for and remove the from the 
-# CMAKE_CUDA_FLAGS so that they are not applied globally.
-#
 if(VLLM_GPU_LANG STREQUAL "CUDA")
+  #
+  # For cuda we want to be able to control which architectures we compile for on 
+  # a per-file basis in order to cut down on compile time. So here we extract
+  # the set of architectures we want to compile for and remove the from the 
+  # CMAKE_CUDA_FLAGS so that they are not applied globally.
+  #
```

Comment:
```suggestion
  #
  # For cuda we want to be able to control which architectures we compile for on 
  # a per-file basis in order to cut down on compile time. So here we extract
  # the set of architectures we want to compile for and remove them from the 
  # CMAKE_CUDA_FLAGS so that they are not applied globally.
  #
```